hack/generator/pkg/astbuilder/literals.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astbuilder
+
+import (
+	"go/ast"
+	"go/token"
+)
+
+// LiteralString() creates the AST node for a literal string value
+func LiteralString(content string) ast.Expr {
```

Comment:
Could use this in a few places that are just doing it the old way? Search for `BasicLit` usage.

Reply:
All the existing usage of `BasicLit` could be replaced. 😁